engine/src/flutter/lib/web_ui/lib/src/engine/lazy_path.dart

```diff
@@ -634,22 +630,34 @@ class LazyPathMetricIterator implements Iterator<ui.PathMetric>, Collectable {
   int _nextIndex = 0;
   DisposablePathMetricIterator? _cachedIterator;
   final List<DisposablePathMetric> _metrics = [];
+  bool _isAtEnd = false;
 
   @override
   ui.PathMetric get current {
-    assert(_nextIndex > 0);
+    if (_nextIndex == 0 || _isAtEnd) {
```

Comment:
That's a BIG string to dump into every flutter app.

Could we do assert silly to wrap this up? Likely to happen in production?

Reply:
Someone *will* change a punctuation or a space in one of the strings and prevent the deduplication 😅